apps/server/src/modules/authentication/controller/account.controller.ts

```diff
@@ -1,20 +1,104 @@
+	}
+
+	// IMPORTANT!!!
+	// updateMyAccount has to occur before updateAccountById, because Nest.js
+	// will always use the first path match and me will be treated as a path parameter
+	// (some e2e tests might fail, if the method order is changed)
+	@Patch('me')
+	@ApiOperation({ summary: 'Updates an account for the authenticated user.' })
+	@ApiResponse({ status: 200, description: 'Account was successfully updated.' })
+	@ApiResponse({ status: 400, type: ValidationError, description: 'Request data has invalid format.' })
+	@ApiResponse({ status: 403, type: ForbiddenOperationError, description: 'Invalid password.' })
+	@ApiResponse({ status: 404, type: EntityNotFoundError, description: 'Account not found.' })
+	async updateMyAccount(@CurrentUser() currentUser: ICurrentUser, @Body() params: PatchMyAccountParams): Promise<void> {
+		return this.accountUc.updateMyAccount(currentUser.userId, params);
+	}
+
+	@Patch(':id')
+	@ApiOperation({ summary: 'Updates an account with given id. Superhero role is REQUIRED.' })
+	@ApiResponse({ status: 200, type: AccountByIdResponse, description: 'Returns updated account.' })
+	@ApiResponse({ status: 400, type: ValidationError, description: 'Request data has invalid format.' })
+	@ApiResponse({ status: 403, type: ForbiddenOperationError, description: 'User is not a superhero.' })
+	@ApiResponse({ status: 404, type: EntityNotFoundError, description: 'Account not found.' })
+	async updateAccountById(
+		@CurrentUser() currentUser: ICurrentUser,
+		@Param() params: AccountByIdParams,
+		@Body() body: AccountByIdBodyParams
+	): Promise<AccountByIdResponse> {
+		return this.accountUc.updateAccountById(currentUser, params, body);
+	}
+
+	@Delete(':id')
+	@ApiOperation({ summary: 'Deletes an account with given id. Superhero role is REQUIRED.' })
+	@ApiResponse({ status: 200, type: AccountByIdResponse, description: 'Returns deleted account.' })
+	@ApiResponse({ status: 400, type: ValidationError, description: 'Request data has invalid format.' })
+	@ApiResponse({ status: 403, type: ForbiddenOperationError, description: 'User is not a superhero.' })
+	@ApiResponse({ status: 404, type: EntityNotFoundError, description: 'Account not found.' })
+	async deleteAccountById(
+		@CurrentUser() currentUser: ICurrentUser,
+		@Param() params: AccountByIdParams
+	): Promise<AccountByIdResponse> {
+		return this.accountUc.deleteAccountById(currentUser, params);
+	}
+
 	@Patch(':id/pw')
+	@ApiOperation({ summary: 'Updates the password of an account with given id.' })
+	@ApiResponse({ status: 200, description: 'Updated password successfully.' })
+	@ApiResponse({ status: 400, type: ValidationError, description: 'Request data has invalid format.' })
+	@ApiResponse({
+		status: 403,
+		type: ForbiddenOperationError,
+		description: 'No permission to change the user password.',
+	})
+	@ApiResponse({ status: 404, type: EntityNotFoundError, description: 'Account or user not found.' })
 	async changePassword(
```

Comment:
As discussed, we probably should remove this and use ` updateAccountById()` instead.